tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -349,10 +352,14 @@ def encode_deltas(
 class PositionList(Generic[EltT]):
     get_range: Callable[[EltT], Range]
     elts: List[EltT] = field(default_factory=list)
+    sort_by_end: bool = False
 
     def sort(self):
         self.elts.sort(key=lambda x: self.get_range(x).start)
 
+    def reverse(self):
+        self.elts.reverse()
+
```

Comment:
This is never used and I'm glad that it isn't because it would certainly break the various `bisect`-based things. `sort_by_end` is also not used as far as I can tell.